Makefile

```diff
@@ -121,12 +121,20 @@ test/pykanidm/mypy:
 test/pykanidm: ## run the test suite (mypy/pylint/pytest) for the kanidm python module
 test/pykanidm: test/pykanidm/pytest test/pykanidm/mypy test/pykanidm/pylint
 
+.PHONY: test/doc/format
+test/doc/format: ## Format docs and the Kanidm book
+	find . -type f -name \*.md -print0 | xargs -r -0 deno fmt --check
```

Comment:
This doesn't need xargs and the problems that it can bring - `find . -type f -name \*.md -exec deno fmt --check "{}" \;` would do.